cmd/otelinmemexporter/store.go

```diff
@@ -155,37 +157,45 @@ func (s *Store) Add(ld pmetric.Metrics) {
 
 // GetAll returns all the aggregated values for all the configured
 // aggregation configs.
-func (s *Store) GetAll() map[string]float64 {
+func (s *Store) GetAll() map[string]map[string]float64 {
 	s.RLock()
 	defer s.RUnlock()
 
-	m := make(map[string]float64, len(s.nums))
+	m := make(map[string]map[string]float64, len(s.nums))
 	for key, cfg := range s.keyM {
-		dp, ok := s.nums[key]
+		dpByGrp, ok := s.nums[key]
 		if !ok {
-			m[key] = 0
+			m[key][""] = 0
```

Comment:
This looks like this can panic when `m[key]` is `nil` (which would make sense as `m[key]` is instantiated just below)

Reply:
Thanks for catching this, it slipped through during refactoring 🤦 . I have fixed it and also updated the test case to catch this.